doc/whats_new.rst

```diff
@@ -139,6 +139,8 @@ API
 
     - Added :class:`mne.decoding.UnsupervisedSpatialFilter` providing interface for scikit-learn decomposition algorithms to be used with MNE data, by `Jean-Remi King`_ and `Asish Panda`_
 
+    - class:`mne.decoding.FilterEstimator` has been deprecated in favor of :class:`mne.decoding.Filterer` by `Asish Panda`_
```

Comment:
maybe I'm missing what exactly the changes are. The `whats_new` just talks about a new class ... is it just the signature which is changing?